syntax = "proto3";

package healthcheck;

service HealthCheck {
  rpc SendHealthCheck (HealthCheckMessage) returns (HealthCheckMessage) {}
}

message HealthCheckMessage {
  string message = 1;
  string server = 2;
}